shares/shares.go

```
@@ -90,33 +72,38 @@ func (s Share) IsCompactShare() (bool, error) {
 	return isCompact, nil
 }
 
+// GetSigner returns the signer of the share, if the
+// share is not of type v1 and is not the first share in a sequence
+// it returns nil
+func GetSigner(share Share) []byte {
+	infoByte := share.InfoByte()
+	if infoByte.Version() != ShareVersionOne {
+		return nil
```

why not return an error?